java/ingest/ingest-core/src/test/java/sleeper/ingest/job/status/WriteToMemoryIngestJobStatusStore.java

```diff
@@ -34,10 +34,26 @@
 public class WriteToMemoryIngestJobStatusStore implements IngestJobStatusStore {
     private final Map<String, TableJobs> tableNameToJobs = new HashMap<>();
 
+    @Override
+    public void jobValidated(String taskId, IngestJob job, Instant validationTime, ValidationData validationData) {
+        ProcessStatusUpdateRecord validationRecord = new ProcessStatusUpdateRecord(job.getId(), null,
+                ValidationStatus.builder().validationTime(validationTime)
+                        .validationData(validationData).build(), taskId);
+        tableNameToJobs.computeIfAbsent(job.getTableName(), tableName -> new TableJobs())
+                .jobIdToUpdateRecords.computeIfAbsent(job.getId(), jobId -> new ArrayList<>())
+                .add(validationRecord);
+    }
+
     @Override
     public void jobStarted(String taskId, IngestJob job, Instant startTime) {
+        jobStarted(taskId, job, startTime, false);
+    }
+
+    @Override
+    public void jobStarted(String taskId, IngestJob job, Instant startTime, boolean validation) {
```

Comment:
Can we have a method named by its purpose rather than passing the validation flag in? It's hard to track when the flag is negated.